draft-ietf-quic-transport.md

```diff
@@ -3571,6 +3571,9 @@ non-ack-eliciting packets need to be acknowledged, an endpoint MAY wait until an
 ack-eliciting packet has been received to include an ACK frame with outgoing
 frames.
```

Comment:
This is pre-existing, but this sentence is weird, particularly in the context of the "An endpoint MUST NOT send a non-ack-eliciting packet in response to a non-ack-eliciting packet..." above.

Is this supposed to be ", and endpoint waits until..." without the MAY?

Reply:
or "can".  I don't think that there is any use for the MAY, but I don't think that the sentence is inherently *bad*.

Reply:
Thanks, I think I'd rather not change it in this PR, but wanted to point out the awkwardness to reviewers.